README.md

```diff
@@ -100,6 +100,10 @@ sudo apt-get install rune shim-rune sgx-tools
 
 Inclavare Containers can be integrated with dockerd and containerd.
 
+- Inclavare Containers integrated with dockerd can be deployed on docker scenarios, you need to install enclave image into container image so that various enclave runtimes can be launched by `rune` through well-defined [Enclave Runtime PAL API](rune/libenclave/internal/runtime/pal/spec.md).
+
+- Inclavare Containers integrated with containerd can be deployed on Confidential Computing Kubernetes Cluster scenarios. Because `shim-rune` and `rune` can compose a basic enclave containerization stack, container image don't need to contain enclave runtime.
```

Comment:
Attentation your spelling please, you should use `container image doesn't` or `container images don't`, You should check grammars before committing your code.

Reply:
You can use [grammarly](https://www.grammarly.com/) to check your spelling before committing your docs.